CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# [1.3.1](https://github.com/princjef/irontask/compare/v1.3.0...v1.3.1) (2020-09-14)
```

Comment:
same things here about reverting changelog and version bump as the other PR. the build system takes care of all of that :)